policy-report/crd/wgpolicyk8s.io_clusterpolicyreports.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.6
+    controller-gen.kubebuilder.io/version: v0.2.5
```

Comment:
@JimBugwadia does it matter that the controller-gen used is over a year old?

Reply:
Update to 2.6